deploy/helm/kuberhealthy/values.yaml

```diff
@@ -90,13 +98,16 @@ check:
     nodeSelector: {}
   podStatus:
     enabled: false
+    runInterval: 5m
+    timeout: 15m
     image:
       repository: kuberhealthy/pod-status-check
       tag: v1.2.2
     extraEnvs:
     nodeSelector: {}
 
 checkReaper:
+  enabled: true
```

Comment:
Mentioned already in the above comment, but a warning would be nice here if this is disabled! Also wondering if this should be contingent on what Kuberhealthy version they're using. I say that if they're using Kuberhealthy 2.1.0+, this should be enabled!